### DISCUSSION TITLE
osu!lazer scores pinnable on the website

### DISCUSSION THREAD

**Discussion by camellirite** [ppy/osu#20466] [Ideas] [answered]
Let me know if should be posted on osu-web but if the anti-cheat system has started processing osu!lazer scores I'd appreciate if they could be pinned on the user profile. I know pp isn't implemented yet and I know they probably won't ever reward pp. The reason is I play 100% on lazer and I miss *something* to work towards, like getting cool score to show off on my profile. 

On lazer
![osu!_BvY59Axg9G](https://user-images.githubusercontent.com/23125409/192139047-24780b0c-ef09-492d-9f86-73be57965dee.png)

On the website 
![firefox_XERYCFzsHR](https://user-images.githubusercontent.com/23125409/192139087-4a3bd6c2-8793-44a5-9b91-e6feec222c2d.png)

**Comment by peppy** [comment, marked as answer]
It will come in time
